### PR TITLE
[1.5.0] Update install/update/uninstall steps for Helm chart changes

### PR DESCRIPTION
**Pull request by treydock**
Signed-off-by: Trey Dockendorf <tdockendorf@osc.edu>

In response to https://github.com/kyverno/kyverno/pull/2357

For the upgrade steps, I opted for "simplest" approach where do full uninstall/re-install.  There are less simple ways to do upgrade documented in https://github.com/kyverno/kyverno/pull/2357 but wasn't sure if was good to document those steps.

### CODE REVIEW

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -447,29 +445,30 @@ helm search repo kyverno
...
 
 {{% alert title="Note" color="warning" %}}
-Upgrading to Kyverno 1.4.2+ (Helm chart v2.0.2) from a version prior to 1.4.2 (Helm chart v2.0.2) will require extra steps.
-The step to remove CRDs will cause all Kyverno policies to get removed, so a backup must be taken.
+Upgrading to Kyverno 1.5.0+ (Helm chart v2.1.0) from a version between 1.4.2 to 1.4.3 (Helm chart `>=v2.0.2 <v2.1.0`) will require extra steps.
+The step to remove CRDs will cause all Kyverno policies to get removed, so a backup must be taken of policies not added by Helm.
 {{% /alert %}}
 
-Below are the steps to upgrade Kyverno to 1.4.2 from a version prior to 1.4.2. The upgrade to 1.4.2+ requires first removing the old CRDs and letting the Helm chart install the CRDs.
+Below are the steps to upgrade Kyverno to 1.5.0 from 1.4.3. The upgrade to 1.50+ requires first removing the old CRDs CRDs chart.
```
Comment (realshuting) [inline content/en/docs/Installation/_index.md:457]: Duplicated `CRDs`.

Comment (treydock) [reply content/en/docs/Installation/_index.md]: Fixed

**Comment by realshuting** [comment]
@chipzoller - can you please help review?

**Review by realshuting — APPROVED** [review]

File: content/en/docs/Installation/_index.md [hunk 2]
```diff
@@ -447,29 +445,30 @@ helm search repo kyverno
...
 
 {{% alert title="Note" color="warning" %}}
-Upgrading to Kyverno 1.4.2+ (Helm chart v2.0.2) from a version prior to 1.4.2 (Helm chart v2.0.2) will require extra steps.
-The step to remove CRDs will cause all Kyverno policies to get removed, so a backup must be taken.
+Upgrading to Kyverno 1.5.0+ (Helm chart v2.1.0) from a version between 1.4.2 to 1.4.3 (Helm chart `>=v2.0.2 <v2.1.0`) will require extra steps.
+The step to remove CRDs will cause all Kyverno policies to get removed, so a backup must be taken of policies not added by Helm.
 {{% /alert %}}
 
-Below are the steps to upgrade Kyverno to 1.4.2 from a version prior to 1.4.2. The upgrade to 1.4.2+ requires first removing the old CRDs and letting the Helm chart install the CRDs.
+Below are the steps to upgrade Kyverno to 1.5.0 from 1.4.3. The upgrade to 1.50+ requires first removing the old CRDs chart.
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:457]: Second 1.5.0 is missing a period.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Minor comment

File: content/en/docs/Installation/_index.md (hunk 2 above)
Comment (treydock) [reply content/en/docs/Installation/_index.md]: Fixed.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**